projects/libass/build.sh

```diff
@@ -38,14 +38,9 @@ cd $SRC/libass
 
 export PKG_CONFIG_PATH=/work/lib/pkgconfig
 ./autogen.sh
-./configure --disable-asm
-make -j$(nproc)
+./configure FUZZ_CPPFLAGS="-DASS_FUZZMODE=2" --disable-asm --disable-shared --enable-fuzz
+make fuzz/fuzz_ossfuzz
```

Comment:
To continue using parallelisation as before `make -j "$(nproc)" fuzz/fuzz_ossfuzz` would be better.